.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install poetry
+    - name: Build package
+      run: |
+        poetry self add "poetry-dynamic-versioning[plugin]"
+        poetry config repositories.testpypi https://test.pypi.org/legacy/
+        poetry dynamic-versioning --no-cache
+        poetry build
+    - name: Upload wheel to release
+      uses: svenstaro/upload-release-action@v2
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
+        file: dist/*.whl
+        tag: ${{ github.ref }}
+        overwrite: true
+        file_glob: true
+    - name: Deploy to PyPI
+      run: |
+        poetry publish -r testpypi -u "__token__" -p "${{ secrets.TEST_PYPI_TOKEN }}"
+        poetry publish -u "__token__" -p "${{ secrets.POSTGRES_PYPI_TOKEN }}"
```

Poetry hasn't added support for pypi oidc, so let's use the official PyPA option:

```suggestion
    - name: Deploy to PyPI
      uses: pypa/gh-action-pypi-publish@v1.8.5
```

---

Yeah I was going with poetry until we updated and added the PyPI repos. Didn't expect a response in a few hours :O